doc/gui/examples/controls/slider_labels_lov.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+# -----------------------------------------------------------------------------------------
+# To execute this script, make sure that the taipy-gui package is installed in your
+# Python environment and run:
+#     python <script>
+# -----------------------------------------------------------------------------------------
+from taipy.gui import Gui
+
+# Initial value of the slider
+value=0
+
+#If 'labels' is set to True, the slider uses the labels of the 'lov' if there are any.
+page = """
+Labels with lov
+
+<|{value}|slider|lov=0;1;2;3;4;5|labels=True|>
```

Comment:
```suggestion
<|{value}|slider|lov=One;Two;Three;Four;Five|labels|>
```
No need for '=True' in Markdown.
More meaningful labels.